Makefile

```diff
@@ -71,11 +76,15 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
+deploy: NAME_SUFFIX ?= $(USER)
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	cd config/dev && $(KUSTOMIZE) edit set image controller=${IMG}
+	cd config/dev && $(KUSTOMIZE) edit set namesuffix -- -$(NAME_SUFFIX)
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
```

Comment:
So everybody can `make deploy` and that creates resources suffixed with your current `$USER`.